rcl/include/rcl/client.h

```diff
@@ -339,6 +339,11 @@ RCL_WARN_UNUSED
 const rcl_client_options_t *
 rcl_client_get_options(const rcl_client_t * client);
 
+///  Non-safe get_options function.
+/* *INDENT-OFF* */
+#define _client_get_options(client) &client->impl->options;
```

Comment:
Also, there is some inconsistency in that this kind of function is in client, pub, and sub, but not service. Which is strange.